easybuild/framework/easyconfig/default.py

```diff
@@ -172,6 +172,7 @@
     'moduleforceunload': [False, 'Force unload of all modules when loading the extension', MODULES],
     'moduleloadnoconflict': [False, "Don't check for conflicts, unload other versions instead ", MODULES],
     'recursive_module_unload': [False, 'Recursive unload of all dependencies when unloading module', MODULES],
+    'recursive_module_unload_depends_on' : [False, 'Use depends_on (Lmod 7.6.1+) for dependencies in generated module', MODULES],
```

Comment:
@bartoldeman Do you think we need this both as configuration option (`--recursive-module-unload-depends-on`) as well as an easyconfig parameter (`recursive_module_unload_depends_on`)?

What use case do you see for latter, i.e. selectively enabling the use of `depends_on` rather than setting it in the global configuration?

Reply:
i will check... this simply mirrors recursive_module_unload.
If I check the history it looks like the people in Juelich wanted to only have recursive unloading enabled for toolchain modules, and not for other modules, to avoid loadstorms. I will double check how depends_on behaves for us on toolchain modules (we hide them but obviously easybuild uses them itself).